pages/common/supervisorctl.md

```diff
@@ -4,6 +4,10 @@
 > Supervisorctl is the command-line client piece of the supervisor which provides a shell-like interface.
 > More information: <http://supervisord.org>.
 
+- Get status information of all processes:
+
+`supervisorctl status`
```

Comment:
```suggestion
- Show the status of a process (or all processes if `process_name` is not specified):

`supervisorctl status {{process_name}}`
```

Reply:
My original intention is to list available processes with this command, so just 'supervisorctl status' serves the purpose and I prefer simplicity. But  you are free to make changes directly and I am okay with your suggestion. Thank you for your review.

Reply:
> @kbdharun what do you think?

We could separate both examples fas pages can hold a maximum of 8 examples. But I think the current suggestion is fine so will apply it.